service/experiment-requests.yaml

```diff
@@ -32,6 +52,7 @@
     - aflplusplus_explore
     - aflplusplus_explore2exploit
     - aflplusplus_seek
+    - klee
```

Comment:
Can you please remove klee from this line.

Reply:
Done.